capstone/cite/views.py

```diff
@@ -198,10 +198,18 @@ def citation(request, series_slug, volume_number, page_number, case_id=None):
                 .prefetch_related('citations'))
         else:
             cases = []
+        reporters = list(Reporter.objects.filter(short_name_slug=slugify(series_slug)))
```

Comment:
This can be a little cleaner with `first()`, which returns `None` if not found:

```
reporter = Reporter.objects.filter(short_name_slug=slugify(series_slug)).first()
series = reporter.short_name if reporter else series_slug
```